src/main/java/com/minecolonies/api/items/registry/ToolTypeEntry.java

```diff
@@ -0,0 +1,91 @@
+package com.minecolonies.api.items.registry;
+
+import com.minecolonies.api.items.ModToolTypes;
+import com.minecolonies.api.util.constant.IToolType;
+import net.minecraft.network.chat.Component;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.item.ItemStack;
+import net.minecraftforge.registries.RegistryObject;
+
+import java.util.function.Function;
+import java.util.function.Predicate;
+
+public class ToolTypeEntry implements IToolType {
```

Comment:
Javadoc and formatting (entire class)

Reply:
Done. Though the CheckStyle config fails to load for me.